exporter/dynatraceexporter/factory.go

```diff
@@ -77,7 +82,10 @@ func createMetricsExporter(
 		cfg,
 		params.Logger,
 		exp.PushMetricsData,
-		exporterhelper.WithQueue(exporterhelper.DefaultQueueSettings()),
-		exporterhelper.WithRetry(exporterhelper.DefaultRetrySettings()),
+		// explicitly disable timeout since we rely on http.Client timeout logic.
```

Comment:
If this is the case you can simply remove the `TimeoutSettings` from the config. the `timeout` config option will be exposed by the `HttpClientSettings`

Reply:
OK. I followed the example of the splunk and signalfx exporters so it's possible I misunderstood something.

Reply:
Removed in 3ec9e4255e6dd64b88d38fbcf0a01e6708ad05af

Reply:
@bogdandrutu this pattern is also visible in the official `otlphttpexporter` https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/otlphttpexporter/factory.go#L123